### PR TITLE
[PLC] [Refac] Changed some indentation, renamed annotation variables

### PR DESCRIPTION
**Pull request by effectfully**
This

1. makes export lists always indented by 4 spaces. It's diff-friendly, copy-paste-friendly, module-name-changing-friendly and to me it's also eyes-friendly. Also aligns data type declarations in the same way.
2. renames annotations everywhere to `ann`. They were called `a`, `x`, `l`, `pos` and some other names I think. I believe `ann` is more appropriate. I don't like `a`, because sometimes you have some polymorphic functions that deal with annotated things and I want `a` to be reserved for polymorphism
3. renames `NormalizationError` to `NormCheckError`, because the original name never made any sense
4. removes functions needed for handling sizes in Plutus Core
5. splits too long lines into shorter ones

### CODE REVIEW

File: language-plutus-core/test/Spec.hs [hunk 1]
```diff
@@ -188,18 +188,26 @@ testRebindShadowedVariable =
...
+        r1 = TyForall () xName typeKind (TyFun () (TyForall () xName typeKind varX) varX)
+
+        -- (all x (type) (all x (type) (fun x x)))
+        l2 = TyForall () xName typeKind (TyForall () xName typeKind (TyFun () varX varX))
+        -- (all y (type) (all z (type) (fun y z)))
+        r2 = TyForall () yName typeKind (TyForall () zName typeKind (TyFun () varY varZ))
+
     in
-        type0 == type1
+        l1 == r1 && l2 == r2 {- THE LATTER MUST NOT PASS -}
```
Comment (michaelpj) [inline language-plutus-core/test/Spec.hs]: Unrelated changes in here?

Comment (effectfully) [reply language-plutus-core/test/Spec.hs]: Oh, I changed the branch, but I guess did something wrong.
